src/registrar/admin.py

```diff
@@ -1192,7 +1192,7 @@ def custom_election_board(self, obj):
     filter_horizontal = ("current_websites", "alternative_domains", "other_contacts")
 
     # Table ordering
-    ordering = ["requested_domain__name"]
+    ordering = ["-submission_date", "requested_domain__name"]
```

Comment:
Where are the domain requests select2 fields?

Reply:
Does not look like we're using those on domain requests. If that's the case, I think we should make a note in the code somewhere about your guess and leave it alone for now

Reply:
That's a good catch! However, analysts have only read-only access, and I can't think of a scenario where a superadmin has to change the domain request associated with a domain information. In my opinion, any fix here is not worth the overhead we'll introduce. That combo box is also searchable after all, in case there's ever an instance where a superadmin actually needs to do this.

Defer to @abroddrick

Reply:
I completely agree rachid! Not worth the time to update this, nor should we ever change the application assigned to a domain and when in doubt I would NEVER actually even scroll in that drop down so it doesn't matter how it's sorted.